src/controller.coffee

```diff
@@ -827,8 +827,11 @@ Editor::redrawPalette = ->
 
     paletteBlockView.group.element.setAttribute 'data-id', entry.id
 
-    # enable tooltips
-    paletteBlockView.group.element.setAttribute('class', 'droplet-hover-div')
+    # enable tooltips, making sure not to overwrite any additional classes
+    # that may have been added (e.g. by tooltipster)
+    paletteBlockClass = paletteBlockView.group.element.getAttribute('class')
+    unless paletteBlockClass && paletteBlockClass.includes('droplet-hover-div')
+      paletteBlockView.group.element.setAttribute('class', 'droplet-hover-div')
```

Comment:
I think my main concern is that right now this feels like a code.org specific issue (presumably we're the ones using tooltipster to add classes). As much as possible, a fix should address the general case.

It may be the case that we don't actually have to worry about anyone else setting this class name, but it feels cleaner/less destructive to me to have code that says "add this class if we dont already have it" rather than "replace all the classes with this class if we dont have this class"